lang/java/archetypes/avro-service-archetype/src/main/pom/pom.xml

```diff
@@ -79,9 +79,9 @@
       <version>\${logback.version}</version>
     </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>\${junit.version}</version>
+      <groupId>org.junit.vintage</groupId>
+      <artifactId>junit-vintage-engine</artifactId>
+      <version>\${junit5.version}</version>
```

Comment:
This looks like it (only) uses the vintage engine. Do we also want to run JUnit5 (Jupiter) tests?

In that case, we also need to add the Jupiter engine IIRC.

Reply:
Huh -- as far as I remember @opwvhk is correct, and yet JUnit5 tests [are being run](https://github.com/apache/avro/actions/runs/3044499132/jobs/4904953694#step:7:404)!  (TestDataFile is definitely a JUnit5 migrated class).

Reply:
Oh, my mistake, I see -- yes, this archetype can probably be safely left as it is (on JUnit4) or entirely migrate the generated test to [JUnit5](https://github.com/apache/avro/blob/70260919426f89825ca148f5ee815f3b2cf4764d/lang/java/archetypes/avro-service-archetype/src/main/resources/archetype-resources/src/test/java/integration/SimpleOrderServiceIntegrationTest.java#L33-L35) 

Lets not mix JUnit4 and JUnit5 in the archetype, however!

Reply:
Good catch! I will migrate the archetype to JUnit 5.x!

Reply:
Done with https://github.com/apache/avro/pull/1863/commits/65c3ac3354fceaaa6aaa2e81bf54475b1541b2b1